docs/tests/test_twisted_docs.py

```diff
@@ -23,3 +23,32 @@ def test_IPAddress_implementations() -> None:
     with open(BASE_DIR / 'twisted.internet.interfaces.IAddress.html') as stream:
         page = stream.read()
         assert all(impl in page for impl in show_up), page
+
+# Test for https://github.com/twisted/pydoctor/issues/505
+def test_web_template_api() -> None:
+    """
+    This test ensures all important members of the twisted.web.template 
```

Comment:
I am not a big fan of theses tests.
I guess that this should be fine, as Twisted API is kind of fixed and we will not see any big refactoring soon.

But rather than write these tests, I prefer to just run the system tests and make sure no warnings are raised and fail on any warning.

If there is an issue with Twisted upstream I can help to get a fix fast.

Reply:
I think the main issue here is that twisted doesn’t not check for pydoctor warnings.

Reply:
> This is the main reason why I wanted to run Twisted checks inside pydctor and fail on any warnings.

Yes I thought this is what we did in `tox -e twisted-apidoc`